File: RC5SharingSystem/RC5SharingSystem/Utils/Information.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RC5SharingSystem.Utils
{
    public class Information
    {
        public string Path { get; set; }
        public string Contents { get; set; }

        public Information(string path, string contents)
        {
            Path = path;
            Contents = contents;
        }
    }
}

File: CryptoSharingSystem/app/src/main/java/Information.kt
class Information(var path: String, var contents: String) {

    public fun GetHashmap() : HashMap<String, String>
    {
        val map: HashMap<String, String> = hashMapOf("Path" to path, "Contents" to contents)
        return map
    }

    public fun GetJson() : String
    {
        return "{\"path\": \"" + path + "\",\"contents\": \"" + contents + "\"}"
    }
}